kornia/color/rgb.py

```diff
@@ -2,13 +2,58 @@
 import torch.nn as nn
 
 
+class RgbToRgba(nn.Module):
+    r"""Convert image from RGB to RGBA.
+
+    Add an Alpha channel to existing RGB image.
+
+    args:
+        image (torch.Tensor): RGB image to be converted to RGBA.
+        alpha_val (int): An integer between 0 to 255.
+
+    returns:
+        torch.Tensor: RGBA version of the image.
+
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 4, H, W)`
+
+    Examples::
+
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> rgba = kornia.color.RgbToRgba(alpha_val)
+        >>> output = rgba(input)  # 2x4x4x5
+    """
+
+    def __init__(self, alpha_val: float) -> None:
+        super(RgbToRgba, self).__init__()
+        self.alpha_val: float = alpha_val
+
+    def forward(self, image: torch.Tensor, alpha_val: float) -> torch.Tensor:  # type: ignore
+        return rgb_to_rgba(image, self.alpha_val)
+
+
+def rgb_to_rgba(image: torch.Tensor, alpha_val: float) -> torch.Tensor:  # type: ignore
+    # TODO: alpha_val could be a torch.Tensor . Make this a topic for a future PR.
```

Comment:
Maybe there is something I am missing, I think everything should directly work with torch.Tensor.

`
def rgb_to_rgba(image: torch.Tensor, alpha_val: Union[float, torch.Tensor]) -> torch.Tensor:
`

And a check `isinstance(factor, (float, torch.Tensor))` to make sure it is a single value.

Did you mean that each element in the batch can have a different `alpha_val`?

Reply:
Nope. I think the changes you suggested totally make sense. Each element in [,3,4,4] should have same alpha_val I guess. Won't it defeat the purpose of rgba if every element in the batch corresponds to a different A channel with N possible values of alpha_val factor? 
These are just my opinion.

Reply:
[,3,4,4] being an example of a tensor